packages/dx-react-grid-demos/src/demo-sources/grid-featured-data-editing/material-ui/demo.jsx

```diff
@@ -35,9 +35,7 @@ import {
 
 const styles = theme => ({
   lookupEditCell: {
-    paddingTop: theme.spacing.unit * 0.875,
-    paddingRight: theme.spacing.unit,
-    paddingLeft: theme.spacing.unit,
+    padding: theme.spacing(1),
```

Comment:
Are `paddingTop` and `paddingBottom` set to `1` here?

Reply:
On this case, we have removed these paddings to make our markup consistency with previous builds.